singer-io/tap-salesforce
# TDL-15349 Add Date Windowing

# Description of change
 - Add a Date windowing mechanism with a maximum retry of 4 for bulk API call

# QA steps
  - Verify that with pk chunking and with date windowing we get the same sync output in bulk API call
 
# Risks

# Rollback steps
 - revert this branch
